js/client/modules/@arangodb/crash-utils.js

```diff
@@ -172,7 +172,7 @@ function analyzeCoreDumpWindows (instanceInfo) {
 // /        information about the incident.
 // //////////////////////////////////////////////////////////////////////////////
 function analyzeCrash (binary, arangod, options, checkStr) {
-  if (!options.coreCheck) {
+  if (!options.coreCheck || arangod.exitStatus.hasOwnProperty('gdbHint')) {
```

Comment:
Will `arangod.exitStatus` present here in all cases of `analyzeCrash` usage?

Reply:
yes, it should only invoke the crash analysis once it detected the errnous death of a server, or killed it by timeout with SIG_ABRT. This should stop the gdb run from occuring a second time.